pkg/controller/utils/common/common.go

```diff
@@ -3,6 +3,7 @@ package common
 import (
```

Comment:
It's a good point to use seaparate image.go file or/and separate package for this functions, as we will add more functions in #520